manage

```diff
@@ -299,8 +356,14 @@ $app"
   queue="$(echo "$queue" | sed 's/^\( *[^ ]\+\) /\1;/')"
   
   #validate the selections first
-  queue="$(validate_apps_gui "$queue")"
-  [ -z "$queue" ] && exit 0
+  if [[ "$queue" =~ ^"update;filelist:".* ]];then
+    # the queue is being used to pass a list of files to update
+    # skip app validation
+    true
+  else
+    queue="$(validate_apps_gui "$queue")"
+    [ -z "$queue" ] && exit 0
+  fi
```

Comment:
it doesn't
this is under the `if [ "$1" == 'daemon' ];then` where the only input that is ever passed is one line.

https://github.com/Botspot/pi-apps/blob/a2b205e40f8eae27e4affc3c5508a2f6d9671391/manage#L214C1-L366

`"${DIRECTORY}/manage" daemon` is only ever called with one line of input
https://github.com/Botspot/pi-apps/blob/a2b205e40f8eae27e4affc3c5508a2f6d9671391/api#L699
https://github.com/Botspot/pi-apps/blob/a2b205e40f8eae27e4affc3c5508a2f6d9671391/api#L716

Reply:
I would like to change it so that we could pass multiple lines of queue to daemon at once but that would require a refactor of the `terminal_manage` function which, as shown above, is the only function that calls `"${DIRECTORY}/manage" daemon`

Reply:
>Why not rather change validate_apps_gui to skip any lines that include `update;filelist:`?

implemented in https://github.com/Botspot/pi-apps/pull/2340/commits/93c9ed847a6cf8f93f13406cd671196f4e7f1851 even though multiple lines of input can never occur as mentioned above

Reply:
> I would like to change it so that we could pass multiple lines of queue to `"${DIRECTORY}/manage" daemon` at once but that would require a refactor of the `terminal_manage` function which, as shown above, is the only function that calls `"${DIRECTORY}/manage" daemon`. the problem is `terminal_manage` takes two arguments, the first is the action, and the second is the appname.

solved and implemented with
https://github.com/Botspot/pi-apps/pull/2340/commits/b3933ec55d0366c55c94fac74fc148f88c77fbf6
now being used in the updater (for speed)
https://github.com/Botspot/pi-apps/pull/2340/commits/d5b8f386c9c18ac3c84611e638b95ae925e497a0